### ISSUE TITLE
Consider removing the timeline as this component is not accessible

### ISSUE THREAD

**Issue by gagarine** [swiss/styleguide#133]
This component is very hard to use, some of the content is hidden, not nice on mobile, and is slow to load. Lot of user will have hard time to access the content. We should consider remove it and find alternative way to present content in a time context.

FB like timeline or text with using a date in the title are viable alternative.
